bundles/AdminBundle/Resources/public/js/pimcore/event-dispatcher.js

```diff
@@ -1,56 +0,0 @@
-/**
- * Pimcore
- *
- * This source file is available under two different licenses:
- * - GNU General Public License version 3 (GPLv3)
- * - Pimcore Commercial License (PCL)
- * Full copyright and license information is available in
- * LICENSE.md which is distributed with this source code.
- *
- * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
- * @license    http://www.pimcore.org/license     GPLv3 and PCL
- */
-
-pimcore.registerNS("pimcore.eventDispatcher");
-pimcore.eventDispatcher = {
```

Comment:
removing this is a BC break

Reply:
https://github.com/coreshop/CoreShop/pull/2046

Reply:
Thanks for the hint, we didn't expect that someone is using that, because actually it was just intended to be used internally and was not documented.  
We probably need to make that clearer in 11 by using annotations in JS as well.

Reply:
@brusch Berni, CoreShop is using everything Pimcore throws at it ;). You should know that by now :P